com.google.AndroidStudio.json

```
@@ -102,8 +102,8 @@
                 {
                     "type": "archive",
                     "strip-components": 0,
-                    "url": "https://github.com/git-lfs/git-lfs/releases/download/v2.12.0/git-lfs-linux-amd64-v2.12.0.tar.gz",
-                    "sha256": "f9befd0fa0b19517b8ed14ab07812f0d39d776d8c9ea0023e343e30ff300813f"
+                    "url": "https://github.com/git-lfs/git-lfs/releases/download/v2.9.2/git-lfs-linux-amd64-v2.13.2.tar.gz",
```

Git-lfs url is incorrect. Try to change to:
```suggestion
                    "url": "https://github.com/git-lfs/git-lfs/releases/download/v2.13.2/git-lfs-linux-amd64-v2.13.2.tar.gz",
```